README.md

```diff
@@ -7,24 +7,27 @@ information about the tesnet hardfork voting.
 
 ## Developing
 
+
+1. Install Go version 1.11 or higher, and Git.
+Make sure each of these are in the PATH.
+
+2. Clone this repository.
+
+3. Install hardforkdemo
 ``` bash
-go get -v github.com/golang/dep/cmd/dep
```

Comment:
the `go install` is not enough.  there are template files required too.

Reply:
If installing outside GOPATH, `go install` is enough, I believe, isn't it?

Should I add the description if building inside GOPATH, as well?

Reply:
Did you try `$ go install github.com/decred/hardforkdemo`?

Reply:
Yes, I did. If running inside `hardforkdemo` directory, it works.

Does it work for you? What answer does it give?

Reply:
but how does one get that hardforkdemo directory now that the instructions where removed?

Reply:
fair point. Is it better now?